fileattrs/elfoptimized.attr

```diff
@@ -0,0 +1,4 @@
+%__elf_requires		%{_rpmconfigdir}/elfdeps --requires %{?__filter_GLIBC_PRIVATE:--filter-private}
+%__elf_magic		ELF (32|64)-bit.*$
```

Comment:
This line appears to originate from the Clear Linux patched version of elf.attr, but it should probably use the same value as `%__elf_magic` from elf.attr instead.

Reply:
Good catch, will change.